hutch_python/load_conf.py

```diff
@@ -219,6 +224,18 @@ def load_conf(conf, hutch_dir=None):
             bp = get_lightpath(db, hutch)
             cache(**{"{}_beampath".format(hutch.lower()): bp})
 
+    # Elog
+    with safe_load('elog'):
+        # Use the fact if we we used the default_platform or not to decide
+        # whether we are in a specialty station or not
+        if not default_platform:
```

Comment:
Dumb comment warning: the `if not ... else` pattern is my pet peeve because `if ... else` is clearer 99% of the time (just reversing the code blocks). You don't have to change this, I just wanted to mention it.